aim/sdk/repo.py

```diff
@@ -507,17 +522,19 @@ def move_runs(self, run_hashes: List[str], dest_repo: 'Repo') -> Tuple[bool, Lis
         else:
             return True, []
 
-    def query_metrics(self, query: str = '') -> QuerySequenceCollection:
+    def query_metrics(self, query: str = '', report_mode: int = 1) -> QuerySequenceCollection:
         """Get metrics satisfying query expression.
 
         Args:
              query (str): query expression.
+             report_mode(:obj:`str`, optional): indicates report mode (0: disabled, 1: tqdm progress bar, 2: tuple).
+                                                1 if not specified.
         Returns:
             :obj:`MetricCollection`: Iterable for metrics matching query expression.
         """
         self._prepare_runs_cache()
         from aim.sdk.sequences.metric import Metric
-        return QuerySequenceCollection(repo=self, seq_cls=Metric, query=query)
+        return QuerySequenceCollection(repo=self, seq_cls=Metric, query=query, report_mode=report_mode)
```

Comment:
what about `query_images` etc?

Reply:
Oops, forgot about those, cause in APIs those are accessed directly and not via `repo` methods. Will update.